app/src/main/res/values/strings.xml

```diff
@@ -1,7 +1,7 @@
 <resources>
     <!-- APIs and Integrations -->
     <string name="parse_app_id">traveltrails</string>
-    <string name="google_api_key">AIzaSyD4JhCWAAVr5oxzC9dzYQl6ed2OeHXuLdQ</string>
+    <string name="google_api_key">AIzaSyBy3aSmVTrzxJ3ycvpIJ26Jv2PYLQHFYSE</string>
```

Comment:
unfortunately, that api key did not work for me. It gave an error: Status{statusCode=PLACES_API_INVALID_APP, resolution=null}.
That's why I had to create another one.